extensions/ql-vscode/src/cli.ts

```diff
@@ -107,6 +107,20 @@ export type MlModelInfo = {
 /** The expected output of `codeql resolve ml-models`. */
 export type MlModelsInfo = { models: MlModelInfo[] };
 
+export type DataExtensionInfo = {
```

Comment:
Appreciate we use `Info` in other places in the codebase but I think it's good to avoid it - all types carry information. Is this simply a `DataExtension`? Or a `CodeQLDataExtension`?

Reply:
Good point, I'll rename them to match what they are actually returning: `ResolveExtensionsResult` and `DataExtensionResult`. This also matches the names in the CLI.

Reply:
The names still don't quite make sense to me but it might be because of my unfamiliarity with the area. Names that come up in my head are `ExtensionResolutionResult` and `ResolvedExtension`. But see what you prefer.

Reply:
I think `ResolveExtensionsResult` and `DataExtensionResult` make it easier to find the definition of these types in the CLI codebase, and changing it to `ExtensionResolutionResult` will make it harder to match those.